.github/workflows/main.yml

```diff
@@ -83,6 +83,36 @@ jobs:
           RUSTFLAGS: -Ctarget-feature=+crt-static -Cdebuginfo=1
         run: cargo build --features bundled
 
+  test-wasm32-unknown-unknown:
+    name: Test ${{ matrix.os }} (wasm32-unknown-unknown)
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+
+    steps:
+      - uses: actions/checkout@v3
+      # This has a matcher for test panics, so we use it even though elsewhere
+      # we use actions-rs/toolchain.
+      - uses: hecrj/setup-rust-action@v1
+        with:
+          rust-version: stable${{ matrix.host }}
+          targets: ${{ matrix.target }}
+
+      - name: Install Homebrew llvm on macOS
+        if: matrix.os == 'macos-latest'
+        run: brew install llvm
+
+      - name: Add llvm path on Windows
+        if: matrix.os == 'windows-latest'
+        run: echo "C:\msys64\mingw64\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+
+      - run: npm install -g wasm-pack
+
+      - name: Test wasm-pack ${{ matrix.host }}
+        run: wasm-pack test --node --features bundled
```

Comment:
This should have a browser-bound test as well, no? Otherwise there's no way to know in CI if the `js_sys` dependency or another browser-specific one breaks.

Reply:
A browser test is certainly reasonable to add, but note that there are no web dependencies, only Javascript dependencies, which node fulfills just as well.